tests/Feature/ApiTest.php

```diff
@@ -0,0 +1,512 @@
+<?php
+
+namespace Tests\Feature;
+
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
+
+class ApiTest extends TestCase
+{
+    function testGetAllMembers()
+    {
+        $response = $this->json('GET', '/api/members');
```

Comment:
Please confirm you are getting the correct list.

Reply:
This may be a hard this to check how many element when growing members.

Reply:
You have 2 options to prevent that case:
1. You can specify a special database for testing in `.env.testing` and set `APP_ENV = testing`
2. Override the environmental variables using `phpunit.xml` (and specify a special database for testing)

That way you don't have to mix your data for the tests and actual operation.

Reply:
https://laravel.com/docs/5.6/configuration#environment-configuration

Reply:
https://laravel.com/docs/5.6/database-testing#resetting-the-database-after-each-test

Reply:
I have do it, thanks 
i create an element and delete it. It complete a CRUD circle